profiles/dunggramer.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/42514846/155019756-71e38cc6-cd52-45e2-bb2e-11ff542d9652.png",
+  "issueId": 1045,
+  "name": "DungGramer",
+  "username": "DungGramer"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```